webui/README.md

```diff
@@ -53,12 +53,23 @@ The content are described in the template file.
 You will need to change the defaults if you would like to enable authentication or are running the eidos on
 a port other than 8080.
 
-### Running the Server
+### Running the Server for Development
+
+To build for dev mode, run:
+
+```bash
+turbo run build
```

Comment:
pnpm build--turbo is not a command that will be recognized unless they have it installed globally

Reply:
build is defined in turbo.json, which I believe is where this command points to?